client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -103,20 +101,18 @@ const ecommerceFlow: Flow = {
 						from_section: 'default',
 					} );
 
-					setPlanCartItem( { product_slug: selectedPlan } );
+					setPlanCartItem( {
+						product_slug: selectedPlan,
+						extra: { design: selectedDesign?.recipe?.stylesheet },
```

Comment:
@daledupreez mentioned making this value `design` more specific, like `headstart_theme`